tests/integration_tests/clouds.py

```diff
@@ -143,20 +144,45 @@ def _get_cloud_instance(self):
         )
 
 
-class LxdContainerCloud(IntegrationCloud):
-    datasource = 'lxd_container'
-    integration_instance_cls = IntegrationLxdContainerInstance
+class _LxdIntegrationCloud(IntegrationCloud, ABC):
```

Comment:
I would expect this to inherit `ABC` from `IntegrationCloud`; as we aren't adding any abstractmethods ourselves, I don't know if we need to redeclare it here.

Reply:
Sure, it more symbolic as a "hey, this should be thought of an abstract base class so don't implement it directly", but I can remove it.